code/__marcos/subsystems.dm

```diff
@@ -4,9 +4,9 @@ Moved it here from machines subsystem, if to leave there, the macros won't be ac
 
 #define START_PROCESSING_IN_LIST(Datum, List) \
 if (Datum.is_processing) {\
-	if(Datum.is_processing != #Processor)\
```

Comment:
This line is pretty perplexing. What was #Processor before?

Replacing it with a hardcoded string is pretty suspect. It seems even wierder that the string specifically references machines. Things aside from machines can process

Reply:
@NanakoAC if you look at `MC.dm` there's `START_PROCESSING` defines that are similar to this one we're in right now. Except this one is specifically for specific sub-lists of `SSmachines`.

This line now matches the line below which is what it's supposed to be. I believe `#Processor` was copy pasted from `START_PROCESSING` (where it does make sense) and not replaced.